content/impact/impact.Rmd

```diff
@@ -127,15 +127,26 @@ data$cite %>% writeLines()
 
 The AusTraits team has delivered talks about AusTraits at multiple conferences and workshops:
 
-- Wenk et al "The AusTraits database structure and R package: introducing a tool for building ecological trait databases", Ecological Society of Australia, 1 December 2022
-- Wenk et al "Documenting the AusTraits ontology", ARDC Vocabulary Symposium, 15 November 2022
-- Wenk et al "The AusTraits data structure and R package: introducing a tool for building ecological trait databases", Open Traits Network, 1 September 2022
-- Gallagher et al "Plant trait composition at the continental scale: using AusTraits to map trait variation and functional diversity across Australia", Ecological Society of Australia, 23 November 2021
+- Sauquet et al "AusTraits: Australia's plant trait database ", Biosystematics 2023, 28 November 2023.
+- Wenk et al "Building the AusTraits Plant Dictionary", ARDC Vocabulary Symposium, 14 November 2023.
```

Comment:
Falster et al "AusTraits: Why biological collections are key to operationalising trait-based methods", ALA & GBIF Symposium on The Changing Face of Biological Collections, 20 October 2023